src/stateManagement/textStyle.js

```diff
@@ -1,38 +1,56 @@
-let defaultFontSize = 15,
-  defaultFont = `${defaultFontSize}px Arial`,
-  defaultBackgroundColor = 'transparent';
+const textStyle = {
+  fontSize: 15,
+  fontFamily: 'Arial',
+  backgroundColor: 'transparent',
+};
 
+// Deprecate
 function setFont(font) {
-  defaultFont = font;
+  const split = font.split('px ');
+
+  if (split.length === 2) {
+    setFontSize(split[0]);
+    setFontFamily(split[1]);
+  }
 }
 
 function getFont() {
-  return defaultFont;
+  return `${textStyle.fontSize}px ${textStyle.fontFamily}`;
+}
+
+function setFontFamily(fontFamily) {
+  textStyle.fontFamily = fontFamily;
+}
+
+function getFontFamily() {
+  return textStyle.fontFamily;
 }
 
 function setFontSize(fontSize) {
-  defaultFontSize = fontSize;
+  textStyle.fontSize = parseFloat(fontSize);
```

Comment:
Can we do a minimum validation to check if is Number?

Reply:
Added.
Thank you!